app/assets/javascripts/discourse/app/components/topic-list.js

```diff
@@ -73,7 +73,15 @@ export default Component.extend(LoadMore, {
 
     let scrollTo = this.session.get("topicListScrollPosition");
     if (scrollTo && scrollTo >= 0) {
-      schedule("afterRender", () => $(window).scrollTop(scrollTo + 1));
+      schedule("afterRender", () => {
+        debounce(
+          this,
+          function () {
+            $(window).scrollTop(scrollTo + 1);
```

Comment:
Is there any chance this will execute after the component has been destroyed? We typically have to check for that in events like these (especially for test environments.)